### PR TITLE
Adds color options to Chip component

### LINKED ISSUE #386: [BUG] Missing color prop in the Chip component
Just realised that Chips component have also color classes: https://github.com/italia/bootstrap-italia/blob/e123ded4b957a46f4e2af81dc5b4e36f1c4324e6/src/scss/custom/_chips.scss#L165

Classes are as usual:

* `primary`
* `secondary`
* `success`
* `warning`
* `danger`
* `info`

Consider to have a new prop `color` as used in reactstrap components.

### PR DESCRIPTION
**Pull request by Pratyush1197**
Fixes #386 

#### PR Checklist

- [ ] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by francescozaia** [comment]
Hey @Pratyush1197 final result is not yet ok. You can see that text is not perfectly aligned within the _pill_.
Screenshot 2019-10-10 at 12 21 25

I guess you should compare your work to the standard version. At first sight, that's using `chip-simple` to put some padding to the right.

File: src/components/Chips/Chip.js [hunk 1]
```diff
@@ -12,23 +12,27 @@ const propTypes = {
...
+  color: PropTypes.string
 }
 
 const defaultProps = {
   tag: 'div',
   simple: false,
   large: false,
-  disabled: false
+  disabled: false,
+  color: 'chip-primary'
```
Comment (dej611) [inline src/components/Chips/Chip.js]: Just `primary`

File: src/components/Chips/Chip.js [hunk 2]
```diff
@@ -12,23 +12,27 @@ const propTypes = {
...
-    'chip-simple': simple,
-    'chip-lg': large,
-    'chip-disabled': disabled
-  })
+  const { className, color, tag: Tag, simple, large, disabled, ...attributes } = props
+  var classes = classNames('chip', className, 
+  simple ? 'chip-simple' : false,
+  large ? 'chip-lg' : false,
+  disabled ? 'chip-disabled' : false,
+  'chip-' + color
```
Comment (dej611) [inline src/components/Chips/Chip.js]: Can you please restore the old format?

```js
const classes = classNames('chip', className, {
  'chip-simple': simple,
  'chip-lg': large,
  'chip-disabled': disabled,
  [`chip-${color}`]: color
})
```

**Comment by Pratyush1197** [comment]
I have made the changes

**Comment by francescozaia** [comment]
Thanks @Pratyush1197! 🙇

**Pull request merged**